gradle/native_mpp.gradle

```diff
@@ -45,6 +45,12 @@ kotlin {
 
             // WASM
             addTarget(presets.wasm32)
+
+            // Android NDK
+            addTarget(presets.androidNativeArm32)
+            addTarget(presets.androidNativeArm64)
+            addTarget(presets.androidNativeX86)
+            addTarget(presets.androidNativeX64)
```

Comment:
Have you checked that all these targets are being tested on our TeamCity, can be depend upon and are generally valid?

Reply:
It builds locally, and it builds on TeamCity. Not sure if we have any dedicated Gradle tools to test any targets except the host and iOS ones.